modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/tags/Tags.es.js

```diff
@@ -64,24 +96,60 @@ export default withRouter(
 			);
 		};
 
+		const orderByOptions = getOrderByOptions();
+
 		const [debounceCallback] = useDebounceCallback((search) => {
 			setSearch(search);
 		}, 500);
 
 		return (
 			<>
 				<div className="container">
-					<div className="row">
-						<div className="col-md-4 offset-md-8">
+					<div className="d-flex flex-row">
+						<div className="d-flex flex-column flex-grow-1">
+							<ClayInput.Group className="c-mt-3 justify-content-end">
+								<ClayInput.GroupItem shrink>
+									<label
+										className="align-items-center d-inline-flex m-0 text-secondary"
+										htmlFor="tagsOrderBy"
+									>
+										{Liferay.Language.get('order-by')}
+									</label>
+								</ClayInput.GroupItem>
+
+								<ClayInput.GroupItem shrink>
+									<ClaySelect
+										className="bg-transparent border-0"
+										disabled={loading}
+										id="tagsOrderBy"
+										onChange={(event) => {
+											setLoading(true);
+											setOrderBy(event.target.value);
+										}}
+										value={orderBy}
+									>
+										{orderByOptions.map((option) => (
+											<ClaySelect.Option
+												key={option.value}
```

Comment:
here I think you can spread option, do something like {...option} except for the key, damn.